### PR TITLE
OKN-1961

### PR DESCRIPTION
**Pull request by cristain7**
Remove menu list component

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ See README.md for release description.
 
 - organisms-navigation-shape_hero: fixed aligning of main columns for all break points
 
+### Removed
+
+- molecules-navigation-menu_list: removed component
```
Comment (simonerlandsen) [inline CHANGELOG.md]: menu_list.sass is imported in internet. u should check that removing it does not explode the styling on internet

Comment (simonerlandsen) [reply CHANGELOG.md]: it does not seam that u have copied over the style for this component in the internet project

Comment (cristain7) [reply CHANGELOG.md]: Yes, it is copied in include/layouttemplates/global/
I just tested in the internet repo that it works without the component in the node_modules folder and the only thing required is to remove the import in styles.scss. This has to be done when this task is merged into develop and release new version.

**Comment by simonerlandsen** [comment]
ok

**Pull request closed without merging**